kafka-connectors/kafka-common/src/main/java/org/apache/apex/malhar/kafka/AbstractBaseKafkaInputOperator.java

```diff
@@ -77,12 +77,12 @@
  * @since 3.3.0
  */
 @InterfaceStability.Evolving
-public abstract class AbstractKafkaInputOperator implements InputOperator,
+public abstract class AbstractBaseKafkaInputOperator implements InputOperator,
```

Comment:
I don't see any need for the rename. I also don't see the need for derived abstract classes when the only difference is the creation of the consumer (you can add that to KafkaSinglePortInputOperator). Also, existing abstract class was evolving, the derived classes you created are not. Any new classes should be evolving.